src/backend/lib/queue.js

```diff
@@ -1,13 +1,16 @@
 const Bull = require('bull');
-const { Redis, redisUrl } = require('./redis');
+const url = require('url');
+const { Redis, redisUrl, redisPassword } = require('./redis');
 const { logger } = require('../utils/logger');
 
 /**
  * Shared redis connections for pub/sub, see:
  * https://github.com/OptimalBits/bull/blob/28a2b9aa444d028fc5192c9bbdc9bb5811e77b08/PATTERNS.md#reusing-redis-connections
  */
-const client = new Redis(redisUrl);
-const subscriber = new Redis(redisUrl);
+const { port, host } = url.parse(redisUrl, true);
+
+const client = new Redis(port, host, redisPassword);
```

Comment:
Based on the documentation you have addressed for Redis, the third argument passed to the constructor (aka [options]) is an object. So we probably need to pass it like: `{password: redisPassword}`  and actually they have used this example in their document:
```var authedRedis = new Redis(6380, '192.168.100.1', { password: 'password' });```